examples/vl3_basic/vl3-nse/cmd/vl3_connect.go

```diff
@@ -156,9 +158,10 @@ func (vxc *vL3ConnectComposite) processPeerRequest(vl3SrcEndpointName string, re
 func (vxc *vL3ConnectComposite) Request(ctx context.Context,
 	request *networkservice.NetworkServiceRequest) (*connection.Connection, error) {
 	logger := logrus.New() // endpoint.Log(ctx)
+	requestConnection := request.GetConnection()
```

Comment:
conn is more suggestive than requestConnection  as variable name

Reply:
Done